### PR TITLE
[docs] Docs updated for yb_admin reference with rename-id subcommand.

### LINKED ISSUE #13530: [Docs] The "rename_id" subcommand for "yb-admin alter_universe_replication" is undocumented
### Description

A subcommand for renaming xcluster replication streams was added in [10563](https://github.com/yugabyte/yugabyte-db/issues/10563) but was never added to the documentation.

This subcommand should be documented here:
https://docs.yugabyte.com/preview/admin/yb-admin/#alter-universe-replication

Example:

```
yb-admin -master_addresses <target_master_addresses> \
    alter_universe_replication <source_universe_uuid>_<replication_name> \
    rename_id <source_universe_uuid>_<new_replication_name>
```

*target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is localhost:7100.
*source_universe_uuid*: The UUID of the source universe.
*replication_name*: The name of the replication to be altered.
*new_replication_name*: The new name of the replication stream

This subcommand is available starting in version 2.6.

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #13530 

@netlify /preview/admin/yb-admin/

Summary
- included rename-id for versions :preview, stable, 2.12 and 2.8 : https://deploy-preview-13685--infallible-bardeen-164bc9.netlify.app/preview/admin/yb-admin/#setup-universe-replication
- Removed `create-cdc-stream` after I got confirmation from engineering that we don't use it.
- Restructured xCluster Replication commands section which now has the right subcommands(earlier was under [Master follower deployment commands](https://docs.yugabyte.com/v2.12/admin/yb-admin/#master-follower-deployment-commands) which is incorrect): https://deploy-preview-13685--infallible-bardeen-164bc9.netlify.app/v2.12/admin/yb-admin/#xcluster-replication-commands.

### CODE REVIEW

File: docs/content/v2.12/admin/yb-admin.md [hunk 1]
```diff
@@ -508,7 +508,7 @@ Use this command to create custom placement policies only for YCQL tables or tra
 
 #### create_transaction_table
 
-Creates a transaction status table to be used within a region. This command should always be followed by [`modify_table_placement_info`](#modify-table-placement-info) to set the placement information for the newly created transaction status table.
+Creates a transaction status table to be used in a region. This command should always be followed by [`modify_table_placement_info`](#modify-table-placement-info) to set the placement information for the newly created transaction status table.
```
Comment (lizayugabyte) [inline docs/content/v2.12/admin/yb-admin.md:511]: newly-created

File: docs/content/v2.12/admin/yb-admin.md [hunk 2]
```diff
@@ -1618,6 +1469,159 @@ The CDC bootstrap ids are the ones that should be used with [`setup_universe_rep
...
+
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
+* *source_universe_uuid*: The UUID of the source universe.
+* *replication_name*: The name for the replication.
+* *source_master_addresses*: Comma-separated list of the source master addresses.
+* *comma_separated_list_of_table_ids*: Comma-separated list of source universe table identifiers (`table_id`).
+* *comma_separated_list_of_producer_bootstrap_ids*: Comma-separated list of source universe bootstrap identifiers (`bootstrap_id`). Obtain these with [bootstrap_cdc_producer](#bootstrap-cdc-producer-comma-separated-list-of-table-ids), using a comma-separated list of source universe table IDs.
+
+{{< warning title="Important" >}}
+Enter the source universe bootstrap_ids in the same order as their corresponding table_ids.
```
Comment (lizayugabyte) [inline docs/content/v2.12/admin/yb-admin.md:1500]: Suggestion: 
Enter the source universe bootstrap IDs in the same order as their corresponding table IDs.

File: docs/content/v2.12/admin/yb-admin.md [hunk 3]
```diff
@@ -1618,6 +1469,159 @@ The CDC bootstrap ids are the ones that should be used with [`setup_universe_rep
...
+{{< /warning >}}
+
+{{< note title="Tip" >}}
```
Comment (lizayugabyte) [inline docs/content/v2.12/admin/yb-admin.md:1503]: Suggestion: delete the tip wrapper (just leave the sentence).

File: docs/content/v2.12/admin/yb-admin.md [hunk 4]
```diff
@@ -1618,6 +1469,159 @@ The CDC bootstrap ids are the ones that should be used with [`setup_universe_rep
...
+./bin/yb-admin \
+    -master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 \
+    setup_universe_replication e260b8b6-e89f-4505-bb8e-b31f74aa29f3 \
+    127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
+    000030a5000030008000000000004000,000030a5000030008000000000004005,dfef757c415c4b2cacc9315b8acb539a
+```
+
+#### alter_universe_replication
+
+Changes the universe replication for the specified source universe. Use this command to:
```
Comment (lizayugabyte) [inline docs/content/v2.12/admin/yb-admin.md:1521]: Use this command to do the following:

File: docs/content/v2.12/admin/yb-admin.md [hunk 5]
```diff
@@ -1618,6 +1469,159 @@ The CDC bootstrap ids are the ones that should be used with [`setup_universe_rep
...
+
+* Add or remove tables in an existing replication UUID
```
Comment (lizayugabyte) [inline docs/content/v2.12/admin/yb-admin.md:1523]: Periods are missing at the end of sentences.

File: docs/content/v2.12/admin/yb-admin.md [hunk 6]
```diff
@@ -1618,6 +1469,159 @@ The CDC bootstrap ids are the ones that should be used with [`setup_universe_rep
...
+```
+
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
+* *source_universe_uuid*: The UUID of the source universe.
+* *replication_name*: The name of the replication to be altered.
+* *comma_separated_list_of_table_ids*: Comma-separated list of source universe table identifiers (`table_id`).
+* *comma_separated_list_of_producer_bootstrap_ids*: Comma-separated list of source universe bootstrap identifiers (`bootstrap_id`). Obtain these with [bootstrap_cdc_producer](#bootstrap-cdc-producer-comma-separated-list-of-table-ids), using a comma-separated list of source universe table IDs.
+
+{{< warning title="Important" >}}
+Enter the source universe bootstrap_ids in the same order as their corresponding table_ids.
```
Comment (lizayugabyte) [inline docs/content/v2.12/admin/yb-admin.md:1561]: Enter the source universe bootstrap IDs in the same order as their corresponding table IDs.

File: docs/content/v2.8/admin/yb-admin.md [hunk 7]
```diff
@@ -1446,57 +1335,191 @@ $ ./bin/yb-admin \
...
+---
+
 ### Change data capture (CDC) commands
 
-#### create_cdc_stream
+#### list_cdc_streams
+
+Lists the CDC streams for the specified YB-Master servers.
+
+{{< note title="Tip" >}}
```
Comment (lizayugabyte) [inline docs/content/v2.8/admin/yb-admin.md:1346]: Suggestion: get rid of the tip wrapper (just leave the paragraph).

File: docs/content/v2.8/admin/yb-admin.md [hunk 8]
```diff
@@ -1446,57 +1335,191 @@ $ ./bin/yb-admin \
...
+
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
+* *source_universe_uuid*: The UUID of the source universe.
+* *replication_name*: The name for the replication.
+* *source_master_addresses*: Comma-separated list of the source master addresses.
+* *comma_separated_list_of_table_ids*: Comma-separated list of source universe table identifiers (`table_id`).
+* *comma_separated_list_of_producer_bootstrap_ids*: Comma-separated list of source universe bootstrap identifiers (`bootstrap_id`). Obtain these with [bootstrap_cdc_producer](#bootstrap-cdc-producer-comma-separated-list-of-table-ids), using a comma-separated list of source universe table IDs.
+
+{{< warning title="Important" >}}
+Enter the source universe bootstrap_ids in the same order as their corresponding table_ids.
```
Comment (lizayugabyte) [inline docs/content/v2.8/admin/yb-admin.md:1400]: Enter the source universe bootstrap IDs in the same order as their corresponding table IDs.

File: docs/content/v2.8/admin/yb-admin.md [hunk 9]
```diff
@@ -1446,57 +1335,191 @@ $ ./bin/yb-admin \
...
+{{< /warning >}}
 
 {{< note title="Tip" >}}
```
Comment (lizayugabyte) [inline docs/content/v2.8/admin/yb-admin.md:1403]: Suggestion: get rid of the tip wrapper

File: docs/content/v2.8/admin/yb-admin.md [hunk 10]
```diff
@@ -1446,57 +1335,191 @@ $ ./bin/yb-admin \
...
+    setup_universe_replication e260b8b6-e89f-4505-bb8e-b31f74aa29f3 \
+    127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
+    000030a5000030008000000000004000,000030a5000030008000000000004005,dfef757c415c4b2cacc9315b8acb539a
+```
 
-{{< note title="Tip" >}}
+#### alter_universe_replication
 
-Use this command when setting up universe replication to verify if any tables are configured for replication. If not, run [`setup_universe_replication`](#setup-universe-replication); if tables are already configured for replication, use [`alter_universe_replication`](#alter-universe-replication) to add more tables.
+Changes the universe replication for the specified source universe. Use this command to:
```
Comment (lizayugabyte) [inline docs/content/v2.8/admin/yb-admin.md:1421]: Use this command to do the following:

File: docs/content/v2.8/admin/yb-admin.md [hunk 11]
```diff
@@ -1446,57 +1335,191 @@ $ ./bin/yb-admin \
...
 
-{{< /note >}}
+* Add or remove tables in an existing replication UUID
```
Comment (lizayugabyte) [inline docs/content/v2.8/admin/yb-admin.md:1423]: Missing period at the end of sentences

File: docs/content/v2.8/admin/yb-admin.md [hunk 12]
```diff
@@ -1446,57 +1335,191 @@ $ ./bin/yb-admin \
...
+```
+
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
+* *source_universe_uuid*: The UUID of the source universe.
+* *replication_name*: The name of the replication to be altered.
+* *comma_separated_list_of_table_ids*: Comma-separated list of source universe table identifiers (`table_id`).
+* *comma_separated_list_of_producer_bootstrap_ids*: Comma-separated list of source universe bootstrap identifiers (`bootstrap_id`). Obtain these with [bootstrap_cdc_producer](#bootstrap-cdc-producer-comma-separated-list-of-table-ids), using a comma-separated list of source universe table IDs.
+
+{{< warning title="Important" >}}
+Enter the source universe bootstrap_ids in the same order as their corresponding table_ids.
```
Comment (lizayugabyte) [inline docs/content/v2.8/admin/yb-admin.md:1461]: Enter the source universe bootstrap IDs in the same order as their corresponding table IDs.

**Review by lizayugabyte — COMMENTED** [review]
Minor comments

**Comment by aishwarya24** [comment]
@lizayugabyte - addressed your comments. Thanks for the review!

**Review by ionthegeek — APPROVED** [review]
Looks good.

**Review by lizayugabyte — APPROVED** [review]
LGTM

**Pull request merged**